src/DropdownFilter.webmodeler.ts

```diff
@@ -0,0 +1,8 @@
+import { Component, createElement } from "react";
+
+// tslint:disable-next-line class-name
+export class preview extends Component<{}, {}> {
+    render() {
+        return createElement("div");
+    }
+}
```

Comment:
Rules for conditional visibility

Filter by Attributes
- show attributes, Filter method, value
- hide xpath constraint

Filter by Attributes
- hide attributes, Filter method, value
-show: constraint

- in webmodeler always hide the name

Reply:
Done